cmake/vita_buildprops.cmake

```diff
@@ -0,0 +1,112 @@
+message("Set CMAKE Flags for Vita.")
+include("$ENV{VITASDK}/share/vita.cmake" REQUIRED)
+
+cmake_policy(SET CMP0077 OLD)
+
+if(NOT VITA_APP_NAME)
+    set(VITA_APP_NAME "wip-vita-master test1")
+endif()
+
+if(NOT VITA_TITLEID)
+    set(VITA_TITLEID "THEXTECH2")
+endif()
+
+if(NOT VITA_VERSION)
+    set(VITA_VERSION "01.00")
+endif()
+
+if(NOT VITA_MKSFOEX_FLAGS)
+    set(VITA_MKSFOEX_FLAGS "-d ATTRIBUTE2=12") # ATTRIBUTE2=12 specifies we need more RAM.  
+endif()
+
+
+set(VITA_ADDTL_LIBS
+    FLAC
+    modplug
+    mad
+    opusfile
+    opus
+    vorbisfile
+    vorbis
+    ogg
+    jpeg
+    mpg123
+    debugnet
+    mathneon
+    SceCtrl_stub
+    SceMotion_stub
+    SceHid_stub
+    SceRtc_stub
+    SceNetCtl_stub
+    SceNet_stub
+    SceLibKernel_stub
+    ScePvf_stub
+    SceAppMgr_stub
+    SceAppUtil_stub
+    ScePgf_stub
+    freetype
+    png
+    jpeg
+    SceCommonDialog_stub
+    m
+    zip
+    z
+    pthread
+    SceGxm_stub
+    SceDisplay_stub
+    SceSysmodule_stub
+    SceTouch_stub
+    SceAudio_stub
+    vitashark
+    SceShaccCg_stub
+    SceSysmem_stub
+    SceIofilemgr_stub
+    SceKernelThreadMgr_stub
+    SceKernelDmacMgr_stub
+    SceVshBridge_stub
+)
+
+set(VITA_CMAKE_FLAGS
+    # General/TheXTech
+    "-DVITA=1"
+    "-DENABLE_FPIC=0"
+    "-DBUILD_OGG_VORBIS=1"
+    "-DPGE_NO_THREADING=1"
```

Comment:
This isn’t consistent with your clause in the main CMakeLists file. Additionally, even the ones that are consistent duplicate that file as far as I’m aware.